src/components/Search/SearchButton.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from "react"
-import { useTranslation } from "react-i18next"
```

Comment:
??? This was still using react-i18next?

Reply:
uhmm yes! and it created a hydration issue. It may have been an automated import